### PR TITLE
#283, #284, #286 to prod

### PR DESCRIPTION
**Pull request by jhamman**
- finish off scheduling work for all hubs
- bump dask-kubernetes version
- clean up hub.pangeo.io image

ping @scottyhq, @rabernat, @bartnijssen - I'll merge tomorrow if I don't hear any objections by the afternoon.

### CODE REVIEW

File: pangeo-deploy/values.yaml [hunk 1]
```diff
@@ -37,10 +37,12 @@ pangeo:
         enabled: false
 
     scheduling:
-        userScheduler:
-          enabled: false
-        userPlaceholder:
-          enabled: false
+      userScheduler:
```
Comment (scottyhq) [inline pangeo-deploy/values.yaml:40]: Can you describe the effect of these changes or link to an issue?

File: deployments/nasa/image/binder/environment.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ dependencies:
   - h5netcdf=0.7
   - dask=1.1
```
Comment (scottyhq) [inline deployments/nasa/image/binder/environment.yml:20]: for deployments using `onbuild` we could also remove any packages that are the same as pangeo-stacks/base-notebook or pangeo-notebook

**Review by scottyhq — COMMENTED** [review]
looks good Joe! feel free to merge. just added a couple comments. Also, with the notebooks in dev deployment, should we keep them in a separate repo? When people log in they can just do `git clone dev-notebook-repo` or it could be added as a startup feature in jupterhub config.

File: deployments/nasa/image/binder/environment.yml (hunk 2 above)
Comment (jhamman) [reply deployments/nasa/image/binder/environment.yml:20]: yes, let's leave that for a follow up PR.

File: pangeo-deploy/values.yaml (hunk 1 above)
Comment (jhamman) [reply pangeo-deploy/values.yaml:40]: check this out: https://zero-to-jupyterhub.readthedocs.io/en/latest/optimization.html?highlight=scheduling

**Comment by jhamman** [comment]
> Also, with the notebooks in dev deployment, should we keep them in a separate repo?

These were copied (at one point) to https://github.com/pangeo-data/pangeo-example-notebooks

**Comment by jhamman** [comment]
@scottyhq seems to have fixed icesat2 staging, merging now.

**Pull request merged**